src/client/api/v1/get_balance.h

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef __CLIENT_API_V1_BALANCE_H__
+#define __CLIENT_API_V1_BALANCE_H__
+
+#include <stdint.h>
+
+#include "client/client_service.h"
+#include "core/address.h"
+#include "core/types.h"
+
+typedef struct {
+  byte_t addr[IOTA_ADDRESS_BYTES];
+  uint16_t maxResults;
+  uint16_t count;
+  int64_t balance;
```

Comment:
the balance from the node is unsigned int.
```suggestion
  uint64_t balance;
```